lib/TimeoutException.php

```diff
@@ -7,6 +7,6 @@ class TimeoutException extends \Exception {
      * @param string|null $message
      */
     public function __construct(string $message = null) {
-        parent::__construct($message ?: "Promise resolution timed out");
+        parent::__construct($message ?: "Operation timed out");
```

Comment:
Just because it doesn't have to be. It could be used for anything anyone wants it to be used for ...